topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperator.java

```diff
@@ -700,7 +774,7 @@ void enqueue(Handler<Void> event) {
     /** Called when a topic znode is deleted in ZK */
     Future<Void> onTopicDeleted(LogContext logContext, TopicName topicName) {
         return executeWithTopicLockHeld(logContext, topicName,
-            new Reconciliation("onTopicDeleted") {
+            new Reconciliation("onTopicDeleted", Timer.start(metrics.meterRegistry())) {
```

Comment:
I don't really understand the TO much. But do we have to do this in these `onTopic*` methods? Seems a but fragile to me and must be hard to test. Isn't there some method which is calling these where you can set it?